src/announcements/forms.py

```diff
@@ -4,7 +4,8 @@
 from django.utils import timezone
 
 from bootstrap_datepicker_plus.widgets import DateTimePickerInput
-from django_summernote.widgets import SummernoteInplaceWidget
+
+from utilities.widgets import SummernoteInplaceSafeWidget as SummernoteInplaceWidget
```

Comment:
I would prefer to use the new class name to prevent confusion, as opposed to `as SummernoteInplaceWidget`.  It makes the code more clear and easier to read. Explicit is better than implicit =)

Since we will be having two different widget types (safe and unsafe) the code will not be clear which one a form is using just by reading the code, unless you know to check the import statements.